### PR TITLE
save background logs

### PR DESCRIPTION
**Pull request by moshe-kabala**
- Suport multi outputs for verbose log
- Add verbose logger
- Add helper for connecting file to verbose logger
- Use verbose logger in helm
- Move init var to utils and add logs subdirectory
- Add local.Setup for init var dir, verbose logger and connect it into a file
- Use Setup function

### CODE REVIEW

File: cmd/local/upgrade.go [hunk 1]
```diff
@@ -20,15 +20,19 @@ func NewUpgradeCmd() *cobra.Command {
 				return err
 			}
 			ctx := cmd.Context()
-			bgLogger := local.SetupBackgroundLogger("upgrade")
+			close, err := local.Setup("upgrade")
```
Comment (YuvalFeldman) [inline cmd/local/upgrade.go:23]: I feel like local.setup isnt a good name to understand what this does - does it setup the background logger? does it initialize the env?

File: pkg/local/utils.go [hunk 2]
```diff
@@ -64,11 +68,63 @@ func GetLatestImages(useGpu bool) (necessaryImages []string, backgroundImage str
...
+			return err
+		}
+	} else if err != nil {
+		return err
+	}
+
+	return initVarDirSubDir()
+}
+
+func initVarDirSubDir() error {
```
Comment (YuvalFeldman) [inline pkg/local/utils.go:92]: VarDirDubDir?

File: pkg/local/utils.go [hunk 3]
```diff
@@ -64,11 +68,63 @@ func GetLatestImages(useGpu bool) (necessaryImages []string, backgroundImage str
...
+			}
+		} else if err != nil {
+			return err
+		}
+	}
+	return nil
 }
+
+// Setup init VAR_DIR, setup VerboseLog and connect its output into a file
+func Setup(cmdName string) (closeLogFile func(), err error) {
```
Comment (YuvalFeldman) [inline pkg/local/utils.go:109]: As per the previous comment, I would change this name to SetupInfra

File: cmd/local/upgrade.go (hunk 1 above)
Comment (YuvalFeldman) [reply cmd/local/upgrade.go:23]: Looking at the setup func itself I think maybe something like SetupInfra can work here

**Review by YuvalFeldman — APPROVED** [review]
LGTM, see comments

File: pkg/local/utils.go (hunk 2 above)
Comment (moshe-kabala) [reply pkg/local/utils.go]: I change everywhere from Var Dir to StandaloneDir to be more clear

**Pull request merged**